src/controllers/auth.js

```diff
@@ -0,0 +1,83 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const ErrorResponse = require('../utils/errorResponse');
+const User = require('../models/user');
+
+exports.signup = async (req, res, next) => {
+  const { name, surname, emailAddress, schoolName, password, confirmPassword } =
+    req.body;
+
```

Comment:
could u validate if req.body properties is exist or not here

Reply:
Validation added for `/signup`